patches/src/main/kotlin/app/revanced/patches/spotify/misc/fix/SpoofClientPatch.kt

```diff
@@ -118,5 +120,72 @@ val spoofClientPatch = bytecodePatch(
                 """
             )
         }
+
+        firstLoginScreenRenderFingerprint.method.apply {
+            val onEventIndex = indexOfFirstInstructionOrThrow {
+                opcode == Opcode.INVOKE_INTERFACE && getReference<MethodReference>()?.name == "getView"
+            }
+
+            val buttonRegister = getInstruction<OneRegisterInstruction>(onEventIndex + 1).registerA
+
+            addInstruction(
+                onEventIndex + 2,
+                """
+                    invoke-static {v$buttonRegister}, $EXTENSION_CLASS_DESCRIPTOR->setLoginButton(Landroid/view/View;)V
+                    """
+            )
+        }
+
+        secondLoginScreenRenderFingerprint.method.apply {
+            val getViewIndex = indexOfFirstInstructionOrThrow {
+                opcode == Opcode.INVOKE_INTERFACE && getReference<MethodReference>()?.name == "getView"
+            }
+
+            val buttonRegister = getInstruction<OneRegisterInstruction>(getViewIndex + 1).registerA
+
+            // Early returning the render for loop since the first item of the loop is the login button
+            addInstructions(
+                getViewIndex + 2,
+                """
+                    invoke-virtual {v$buttonRegister}, Landroid/view/View;->performClick()Z
+                    return-void
+                    """
+            )
+        }
+
+        thirdLoginScreenRenderFingerprint.method.apply {
+            val invokeSetListenerIndex = indexOfFirstInstructionOrThrow {
+                opcode == Opcode.INVOKE_VIRTUAL &&
+                        getReference<MethodReference>()?.definingClass == "Landroid/view/View;" &&
+                        getReference<MethodReference>()?.name == "setOnClickListener"
+            }
+
+            val buttonRegister = getInstruction<FiveRegisterInstruction>(invokeSetListenerIndex).registerC
+
+            addInstruction(
+                invokeSetListenerIndex + 1,
+                """
+                    invoke-virtual {v$buttonRegister}, Landroid/view/View;->performClick()Z
+                    """
+            )
+        }
+
+        thirdLoginOnClickFingerprint.method.apply {
+            // Return void is NOT at the end of the method
```

Comment:
comments must end with punctuation

```suggestion
            // Return void is NOT at the end of the method.
```